test/specs/sorting.test.js

```diff
@@ -0,0 +1,41 @@
+const { pages } = require('../pages/Pages');
+const users = require('../data/users');
+
+async function verifyNameSorting(order, option) {
+    const sortedNames = await pages.inventoryPage.sortedItemsNames(order);
+    await pages.inventoryPage.selectSortOption(option);
+    const expectedNamesOrder = await pages.inventoryPage.getNames(order);
+    expect(JSON.stringify(sortedNames) === JSON.stringify(expectedNamesOrder)).toBe(true);
+}
+
+async function verifyPriceSorting(order, option) {
+    const sortedPrices = await pages.inventoryPage.sortedItemsPrices(order);
+    await pages.inventoryPage.selectSortOption(option);
+    const expectedPricesOrder = await pages.inventoryPage.getPrices(order);
+    expect(JSON.stringify(sortedPrices) === JSON.stringify(expectedPricesOrder)).toBe(true);
```

Comment:
it's better to compare 2 arrays to equal - https://webdriver.io/docs/api/expect-webdriverio#default-matchers here you'll find that wdio uses Jest matches
and needed matcher is - https://jestjs.io/docs/expect#toequalvalue
the difference is the first one compares strict, while the second one compares `deep` equaly